resolwe/flow/tests/test_env_vars.py

```diff
@@ -12,33 +15,42 @@ class EnvVarsTest(TestCase):
 
     @override_settings(RESOLWE_HOST_URL='some.special.host')
     def test_envvars(self):
-        process = Process.objects.create(
-            name='Test environment variables',
-            requirements={'expression-engine': 'jinja'},
-            contributor=self.contributor,
-            type='test:data:envvars:',
-            input_schema=[],
-            output_schema=[
-                {'name': 'resolweapihost', 'type': 'basic:string:'},
-            ],
-            run={
-                'language': 'bash',
-                'program': """
+        flow_executor = copy.copy(getattr(settings, 'FLOW_EXECUTOR', {}))
```

Comment:
After #220 is merged, this will be much nicer to write with the `@with_custom_executor` decorator.